istioctl/cmd/workload.go

```diff
@@ -222,6 +225,7 @@ Configure requires either the WorkloadGroup artifact path or its location on the
 		" used as the ingress gateway, in the format <service>.<namespace>. If no namespace is provided, the default "+istioNamespace+" namespace will be used.")
 	configureCmd.PersistentFlags().StringVar(&ingressIP, "ingressIP", "", "IP address of the ingress gateway")
 	configureCmd.PersistentFlags().BoolVar(&autoRegister, "autoregister", false, "Creates a WorkloadEntry upon connection to istiod (if enabled in pilot).")
+	configureCmd.PersistentFlags().BoolVar(&dnsCapture, "capture-dns", true, "Enables the capture of outgoing DNS packets on port 53, redirecting to istio-agent")
```

Comment:
I am wondering as we add more and more options do we want this to be done through flags? For this, and certain autoregister, it may make sense. But do we have break-glass for all options?

On pods we have `proxy.istio.io/config` annotation, we should read those off of WG probably if we are not already

Reply:
+1 I'll create an issue.